scripts/aws/setupdb.sh

```diff
@@ -14,3 +14,11 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
+
+# Fetch boundary layer sql files
+FILE_HOST="http://santacruz.internal.azavea.com:9999"
+FILES=("boundary_huc12.sql.gz" "boundary_huc10.sql.gz" "boundary_huc08.sql.gz")
+
+for f in "${FILES[@]}"; do
```

Comment:
Not worth changing, but just wanted to mention that since they increase in steps of 2, you could also loop against `seq`:

```bash
$ seq -f "%02g" 8 2 12
08
10
12
```